modules/telegraf/manifests/init.pp

```diff
@@ -0,0 +1,169 @@
+# == Class: telegraf
```

Comment:
> This overll LGTM!
> 
> @srfraser Hm, I'm a little bit confused about the files under https://github.com/srfraser/build-puppet/tree/telegraf_influxdata/modules/telegraf/log/default. Why do we need them checked-in in the main puppet repo?

This, and a lot of the comments below, are essentially about fixing up someone else's puppet module. I'm not sure how much effort we should spend doing that at the moment, although I can understand the desire to check in as little as needed. What do you think the right balance is?

Reply:
Ah, makes sense then. Since we're hoping to switch off the puppet boat at some point, it's probably not worth investing time for now. You mentioned somewhere that you patched this up to cope with our ancient puppet version so having this for now certainly won't hurt. Feel free to unblock for now. If we need to fix this later, we'll think of a better way 👍